### ISSUE TITLE
Unencrypted Dropbox backup

### ISSUE THREAD

**Issue by patmood** [standardnotes/forum#417]
Is it possible to sync notes to Dropbox un-encrypted? I think I remember it being an option in the really early days, but can't figure it out.

**Comment by mobitar** [comment]
No, this is not possible. The reason is because the server is what syncs to Dropbox, and the server never receives unencrypted data by design.
